README.md

```diff
@@ -54,7 +54,7 @@ go run $GOPATH/src/github.com/openconfig/ygot/generator/generator.go -path=yang
 To allow this file to be auto-created, you can place a command which allows this code generation to be done automatically, either by creating a file within the YANG directory, or directly embedding this command within the source file that populates the structures. For an example, see the `demo/getting_started/main.go` file which includes:
 
 ```
-//go:generate go run ../../generator/generator.go -path=yang -output_file=pkg/ocdemo/oc.go -package_name=ocdemo -generate_fakeroot -fakeroot_name=device -compress_paths=true  -exclude_modules=ietf-interfaces yang/openconfig-interfaces.yang
+//go:generate go run ../../generator/generator.go -path=yang -output_file=pkg/ocdemo/oc.go -package_name=ocdemo -generate_fakeroot -fakeroot_name=device -compress_paths=true -shorten_enum_leaf_names -use_defining_module_for_typedef_enum_names -exclude_modules=ietf-interfaces yang/openconfig-interfaces.yang
```

Comment:
Perhaps we can go with something shorter here? `-typedef_with_defmod` maybe? I think users will need to look up these flags anyway.